docker-compose.yml

```diff
@@ -10,7 +10,7 @@ services:
     image: "core.dnp.dappnode.eth:0.2.42"
     container_name: DAppNodeCore-core.dnp.dappnode.eth
     volumes:
-      - "/usr/src/dappnode/DNCORE/:/usr/src/app/DNCORE/"
+      - "/usr/src/dappnode/:/usr/src/dappnode/"
```

Comment:
Should we bind to `"/usr/src/dappnode/DNCORE:/usr/src/dappnode/DNCORE"` ? why bind to the entire `/usr/src/dappnode` directory?

Reply:
because makes things simple to copy the new host script `dappnode_access_credentials.sh` to the path: `/usr/src/dappnode/scripts`